src/celeritas/phys/PhysicsData.hh

```diff
@@ -256,6 +256,13 @@ struct PhysicsParamsScalars
                    == static_cast<bool>(fixed_step_action));
     }
 
+    //! Lowest kinetic energy for e+/e- energy loss calculation: 1keV
+    using Energy = units::MevEnergy;
+    static CELER_CONSTEXPR_FUNCTION Energy eloss_calc_limit()
+    {
+        return Energy{1e-3};
+    }
```

Comment:
I think it wouldn't be too much of a stretch to make this a parameter settable by `PhysicsParamsOptions` as `MevEnergy`. Could you add it as an input option?

Reply:
As I shortly mentioned, it will break many physics related tests (at least for me using Geant4 11.0 becuase of changes in contents of the Importer) and would like make it configurable after we move to Geant4 11.0 - also, we may never change the default value for the standard EM physics anyway.

Reply:
It can still be an "input option" (a member of `PhysicsParamsOptions`) without being automatically changed by the geant version: I don't see how making it user configurable would change the defaults. I understand that we have lots of hard-coded arbitrary factors in the UrbanMsc classes, but since the other `PhysicsParamsScalars` (derived from `PhysicsParamsOptions`) are also user configurable, I'd like the new value to also be configurable at runtime rather than requiring a change in the source code.